### DISCUSSION TITLE
auto-completion not showing any snippets for org-mode files

### DISCUSSION THREAD

**Discussion by gpagnon** [syl20bnr/spacemacs#16363] [General]
Hi,
I installed the auto-completion layer but when I try to insert a snippet into an org file (SPC-i-s) the helm window does not show any snippets: is there anything else that I should install?

thank you for any suggestion
g,

**Comment by arifer612** [comment]
I believe that it should work out-of-the-box. Can you share the system info (from `spacemacs/report-issue`)?
